ccdproc/combiner.py

```diff
@@ -581,12 +592,23 @@ def combine(img_list, output_file=None,
         ccd.data = ccd.data.astype(dtype)
 
     size_of_an_img = ccd.data.nbytes
-    if ccd.uncertainty is not None:
-        size_of_an_img += ccd.uncertainty.nbytes
+    try:
+        size_of_an_img += ccd.uncertainty.array.nbytes
+    # In case uncertainty is None it has no "array" and in case the "array" is
+    # not a numpy array:
+    except AttributeError:
+        pass
+    # Mask is enforced to be a numpy.array across astropy versions
     if ccd.mask is not None:
         size_of_an_img += ccd.mask.nbytes
-    if ccd.flags is not None:
+    # flags is not necessarily a numpy array so do not fail with an
+    # AttributeError in case something was set!
+    # TODO: Since flags is not supported for combine and read this could
+    #       also be dropped?
```

Comment:
Certainly - but at this point this just calculates the necessary memory for `Combiner` and `Combiner` ignores the flags. Should I remove that comment?